arcgis-ios-sdk-samples/Utility Network/Configure subnetwork trace/README.metadata.json

```diff
@@ -0,0 +1,70 @@
+{
+    "category": "Utility Network",
```

Comment:
It is fine for now, but if #919 merge before this one, then it will need to be updated. I'll double check once it is approved and ready to merge.

Reply:
This need to be fixed.